### ISSUE TITLE
Allow users to use a PetScan query as an input

### ISSUE THREAD

**Issue by Samwalton9** [eggpi/citationhunt#115]
As with https://github.com/eggpi/citationhunt/issues/114, users should be able to use a PetScan query directly to limit the pages Citation Hunt displays.

**Comment by eggpi** [comment]
Hm, if we're thinking of turning the worklist tool into a gateway for other data sources (like PetScan and Pagepile), maybe #114 is the only thing we need here, rather than duplicating functionality between CH and the worklist tool?

That is, a campaign organizer uses the worklist tool to curate the list of interesting articles (via PetScan/Pagepile/whatever), then grabs a link to navigate the list through CH and shares both links with participants. What do you think?

**Comment by Samwalton9** [comment]
I guess my thinking was that people might not need the assigning functionality, and just want to go directly from a list of articles to citation needed tags, without having to go through setting up a worklist and export it to citation hunt.

**Comment by eggpi** [comment]
Yeah, it depend on what we think will be the core use case for event organizers. I would expect them to want to create both a worklist and a CH link (so fixing {{cn}} is a subtask in the worklist, aided by CH), but if we believe bypassing the task management in the worklist tool will be a common case, I'm fine with adding other import method to CH too.

Also, it's worth noting that PetScan and Pagepile are integrated to one another: from PetScan, you can export query results to a Pagepile, and from either Pagepile or PetScan you can import a Pagepile into PetScan to run queries over it. So in theory we could also only support one of the two -- again, depending on how much tool hopping we want to subject event organizers to.
